src/platform/src/rawfile.rs

```diff
@@ -1,20 +1,39 @@
 use core::ops::Deref;
+use sys::{open, dup, close};
```

Comment:
Isn't there a `pub use sys::*` in `lib.rs`?

Reply:
Actually, it turns out there is. https://github.com/redox-os/relibc/blob/master/src/platform/src/lib.rs#L15

So, should I do `platform::{open, dup, close};` instead?